### ISSUE TITLE
different foward results on two machines

### ISSUE THREAD

**Issue by WesleyZhang1991** [dmlc/mxnet#6211]
Hi,all. We managed to train a segmentation network and use it to get predicted results in forward step. The results are the same for several machines.However, on one old machine with cpu i5 + new gpu nvidia1060,the results are different.The data before the foward step is exactly the same and the returned data after network foward is different. No batch norm or dropout is used in the network. Any suggestions?

**Comment by WesleyZhang1991** [comment]
The problem machine produces the same results on its cpu mode and gpu mode. But these results are different from other machines. As we will use the segmentstion for very accurate use, minor differences are not allowed.

**Comment by VoVAllen** [comment]
Have you checked mxnet's version?

**Comment by WesleyZhang1991** [comment]
mxnet version is the same. Also numpy, cv2 and other packages are the same. By the way, we use on Windows platform.

**Comment by piiswrong** [comment]
how did you install it?

**Comment by piiswrong** [comment]
How much is the difference? What hardware does the other machine have?

**Comment by WesleyZhang1991** [comment]
Problem solved.  We test with different models! Though the same name(we renmae the param file), the MD5 is different.
We build windows mxnet on the new machine and put all the necessary dll files to a folder and then set the folder to the system environment path. 
The difference is minor but is visible by human which is not satisfying.
